models/base_diffusion_model.py

```diff
@@ -69,15 +66,27 @@ def __init__(self, opt, rank):
         self.loss_functions_G = ["compute_G_loss_diffusion"]
         self.forward_functions = ["forward_diffusion"]
 
-    def init_semantic_cls(self, opt):
+        if opt.data_refined_mask:
```

Comment:
In `init_semantic_mask` instead ?

Reply:
I don't think so, it's not really for semantic and there's no semantic (yet) in BaseDiffusionModel